include/z64dma.h

```diff
@@ -4,6 +4,18 @@
 #include "ultra64.h"
 #include "alignment.h"
 
+typedef struct {
+    /* 0x00 */ uintptr_t vromStart;
+    /* 0x04 */ uintptr_t vromEnd;
+} RomFile; // size = 0x8
```

Comment:
Given this struct is used a lot outside of z64dma.h context I think it would be a good time to also move RomFile stuff to its own header like `romfile.h`

Reply:
Is it? Don't most use it to make a Dma Request?
(Though I guess most go through wrapper functions)

Reply:
I think the close relationship with dma makes sense, i dunno

Reply:
Yeah anything using RomFile will probably also use other z64dma.h stuff
but not everything seeing the RomFile type (for the purpose of struct layout) will use z64dma.h stuff

this is with header splitting in mind, ideally changing a header causes minimal rebuilding stuff